package io.wiem.exceptions;

public class UserAlreadyExistException extends Throwable {
}
